### PR TITLE
Enable CPU throttling (4.5x)

### PR DESCRIPTION
**Pull request by addyosmani**
This change enables CPU throttling by default and switches the slow-down from 5x -> 3.5x (as discussed with @paulirish in person). The 3.5x aligns closer with the CPU differences observed between a MBP and a Moto G4 (our current recommendation for 'average' mobile hardware).

screen shot 2017-02-25 at 12 13 46 am

If there are any further changes required to enable this, please feel free to 👋

### CODE REVIEW

**Comment by addyosmani** [comment]
cc @wardpeet in case I'm missing anything.

**Comment by ebidel** [comment]
@addyosmani how much closer does this take us to aligning with WPT results? Curious if you had time to run the same site against WPT and this PR?

File: lighthouse-core/gather/gather-runner.js [hunk 1]
```diff
@@ -313,9 +313,8 @@ class GatherRunner {
       return Promise.reject(new Error('You must provide a config'));
     }
 
-    // CPU throttling is temporarily off by default
     if (typeof options.flags.disableCpuThrottling === 'undefined') {
```
Comment (paulirish) [inline lighthouse-core/gather/gather-runner.js]: Afaik you can nuke this conditional

File: lighthouse-extension/app/src/popup.js [hunk 2]
```diff
@@ -124,7 +124,7 @@ function onGenerateReportButtonClick(background, selectedAggregations) {
 
   background.runLighthouseInExtension({
     flags: {
```
Comment (paulirish) [inline lighthouse-extension/app/src/popup.js]: You can drop this flags object

**Comment by paulirish** [comment]
Oh I think there's a note about mobile testing in the readme that can also use a tweak.

**Comment by addyosmani** [comment]
@paulirish Woo. Thanks for the review! I'll get those three bits dropped shortly.

@ebidel Good questions :) The 3.5x number came from locally comparing traces seen on a Moto G4 over remote debugging vs LH on desktop.  Talking to Pat this week, he seemed supportive of the change getting us 90% of the way there in the short term.

I'm going to run some more experiments as a sanity check and report back here.

**Comment by addyosmani** [comment]
@paulirish relevant to the multiplier: do we have any UMA data on the type of desktop/hardware the average DevTools user is on? Just want a sanity check that the MBP is a fair baseline to assume.

**Comment by wardpeet** [comment]
Looks nice 👍 

I'm adding referring this #820 and #747 where we had a previous discussion.

**Comment by addyosmani** [comment]
After trying to look at calibration for a while, I'm updating my recommendation to CPU throttling at 4.5x and tweaking that over time. @paulirish could you let me know if you'd be down with that and I'll update my PR?

Here are a few comparisons between WPT runs over a Moto G4 with 3G and DevTools using a 4.3-4.5x CPU throttling + custom network throttling closer to what WPT is doing:

Polymer Shop - closer with 4.5x CPU slowdown

image

ReactHN - closer with a 4.5x CPU throttling slowdown

image

Fashion United - close..ish with 4.5x CPU slowdown
image

Voice Memos with 4.5x

image

Code.gov with 4.3x CPU slowdown

image

Twitter.com (outlier) at 4x CPU (~1000ms slower)

image

**Comment by addyosmani** [comment]
@paulirish I believe I've addressed your feedback and rebased against master. PTAL whenever you get a chance.

File: lighthouse-cli/bin.ts [hunk 3]
```diff
@@ -124,8 +124,7 @@ Example: --output-path=./lighthouse-results.html`,
   // default values
   .default('chrome-flags', '')
   .default('disable-cpu-throttling', false)
-  .default('output', Printer.GetValidOutputOptions()[Printer.OutputMode.none])
-  .default('output-path', 'stdout')
```
Comment (paulirish) [inline lighthouse-cli/bin.ts]: i don't think you meant to change this, right?

File: lighthouse-cli/bin.ts [hunk 4]
```diff
@@ -124,8 +124,7 @@ Example: --output-path=./lighthouse-results.html`,
   // default values
   .default('chrome-flags', '')
   .default('disable-cpu-throttling', false)
```
Comment (paulirish) [inline lighthouse-cli/bin.ts]: false => true ?

**Pull request merged**